public/locales/ru/translation.json

```
@@ -2,6 +2,8 @@
   "Map": "",
   "Leave Ukraine": "Выехать из Украины",
   "Border info": "Информация о границе",
+  "Login": "логин",
```

In russian "Войти" is commonly used for the login button text

```suggestion
  "Login": "Войти",
```